cpu/esp32/doc.txt

```diff
@@ -1612,7 +1612,7 @@ following configuration parameters have to be defined:
 
 Parameter           | Default   | Description
 :-------------------|:----------|:------------
-#ESP_WIFI_SSID      | "RIOT_AP" | SSID of the AP to be used.
+#WIFI_SSID      | "RIOT_AP" | SSID of the AP to be used.
 ESP_WIFI_EAP_ID     | none      | Optional anonymous identity used in phase 1 (outer) EAP authentication.[1]
 ESP_WIFI_EAP_USER   | none      | User name used in phase 2 (inner) EAP authentication.
 ESP_WIFI_EAP_PASS   | none      | Password used in phase 2 (inner) EAP authentication.
```

Comment:
The question is whether all other WIFI settings that are not specific to ESP should not also be changed.
```suggestion
#WIFI_SSID      | "RIOT_AP" | SSID of the AP to be used.
WIFI_EAP_ID     | none      | Optional anonymous identity used in phase 1 (outer) EAP authentication.[1]
WIFI_EAP_USER   | none      | User name used in phase 2 (inner) EAP authentication.
WIFI_EAP_PASS   | none      | Password used in phase 2 (inner) EAP authentication.
```